supervisely/project/project_meta.py

```diff
@@ -1243,3 +1330,33 @@ def to_detection_task(
                     mapping[obj_class] = None
         res_meta = self.clone(obj_classes=ObjClassCollection(res_classes))
         return res_meta, mapping
+
+    def validate_project_settings(self, add_multi_tag_meta: bool = False) -> ProjectMeta:
+        if self._project_settings.multiview_enabled is True:
+            mtag_name = self._project_settings.multiview_tag_name
+
+            if mtag_name is None:
+                mtag_name = self.get_tag_name(self._project_settings.multiview_tag_id)
+                if mtag_name is None:
+                    return self  # (tag_name, tag_id) == (None, None) is OK
+
+            multi_tag = self.get_tag_meta(mtag_name)
+            if multi_tag is None:
+                if add_multi_tag_meta is False:
+                    raise RuntimeError(
+                        f"The multi-view tag '{mtag_name}' was not found in the '{ProjectMetaJsonFields.TAGS}' field. Please specify the matching tag in the 'meta.json' file."
```

Comment:
The multi-view tag '{mtag_name}' was not found in the project meta. Please directly add the tag meta that will be used for image grouping for multi-view labeling interface.